projects/bp-gallery/src/components/views/start/DailyPicture.tsx

```diff
@@ -73,22 +90,14 @@ const DailyPicture = () => {
   return (
     <div>
       {picture && (
-        <>
-          <h3 className={'text-2xl'}>{t('common.daily-picture')}</h3>
-          <div className={'flex flex-col place-items-center'}>
-            <img
-              className={'w-screen sm:w-auto sm:h-50vh'}
-              src={pictureLink}
-              alt={t('common.daily-picture')}
-              onClick={async () => {
-                await navigateToPicture(picture.id);
-              }}
-            />
-            <div className={'flex flex-col max-w-4xl'}>
-              <p className={'line-clamp-5'}>
-                <h4 className={'text-lg my-1'}>{t('pictureFields.descriptions')}:</h4>
+        <div className='flex justify-center'>
+          <Card className='flex flex-col-reverse md:flex-row rounded-md justify-between max-w-4xl max-h-fit'>
+            <div className='p-4 flex flex-col'>
+              <h3 className={'text-2xl'}>{t('common.daily-picture')}</h3>
+              <div className={'line-clamp-5'}>
```

Comment:
maybe more lines? There's a lot of free space with only 5 lines and most descriptions are probably longer.
![image](https://user-images.githubusercontent.com/64468276/221202117-2425ab2a-b1c0-4316-83bf-88c0b2c75c1f.png)

Reply:
I made it possible to see 10 lines now